class FailedCommand(Exception):
    pass


class UserNotFound(Exception):
    pass
